### DISCUSSION TITLE
Release Schedule

### DISCUSSION THREAD

**Discussion by kwheat2064** [microsoft/openjdk#34] [Q&A] [answered]
I realize this is still in preview, but the AdoptOpenJDK 11.0.11 was released yesterday the 21st. Will there be an updated release of the Microsoft OpenJDK given the new LTS release? The download page is still showing 11.0.10.

**Comment by kwheat2064** [comment, marked as answer]
I see that the latest 11.0.11 binaries are now available on the download page.

**Reply by karianna** [reply]
Our original blog post https://devblogs.microsoft.com/java/announcing-preview-of-microsoft-build-of-openjdk/ has also been updated with an FAQ on this.

**Reply by kwheat2064** [reply]
@karianna Thank you

**Reply by crd477** [reply]
It would be helpful to me if there was a way to be automatically notified of new releases, something like an announcement mailing list, RSS feed, or even just the Github release tagging process.
Thanks!

**Reply by brunoborges** [reply]
@crd477 thanks for the suggestion. We can certainly consider using GitHub Releases in this repo to announce new releases. We will explore that for the January PSU if possible.

**Reply by kwheat2064** [reply]
Is there a tracking page for the January PSU for the Microsoft OpenJDK? Similar to what is available for Adoptium - detailing where in the build process the JDK releases are (TCK/Install build/etc).

**Reply by karianna** [reply]
Hi Kyle, we'll explore if that's possible if/when we move more of our presence to GitHub, but at this stage that's on internal MSFT Azure DevOps.  I appreciate that having to wait not quite knowing when the binaries will land can be frustrating so we'll have a think about how we can communicate that better.

**Reply by kwheat2064** [reply]
@karianna Thank you
